csrc/device_lower/utils.cpp

```diff
@@ -1986,6 +1986,83 @@ std::vector<Expr*> getSyncExprs(AsyncOpType async_type, int64_t keep_stages) {
   return sync_exprs;
 }
 
+std::pair<std::unordered_set<IterDomain*>, std::unordered_set<IterDomain*>>
```

Comment:
Can you please add an example of IDs with MmaOp that are not included in the returned set?